.gitignore

```diff
@@ -6,3 +6,6 @@ values-private.yaml
 
 # Vim swapfiles
 *.sw[po]
+
+# Example temp files
+examples/kubernetes-in-docker/temp*
```

Comment:
Good catch. How did you spot that this was missing?

Reply:
Visual Studio code "greys out" any files or directories mentioned in your gitignore